pyproject.toml

```diff
@@ -10,13 +10,13 @@
 [build-system]
 build-backend = 'mesonpy'
 requires = [
-    "meson-python>=0.5.0",
-    "Cython>=0.29.21",
-    "pybind11>=2.4.3",
-    "pythran>=0.9.12",
+    "meson-python>=0.5.0,<0.6.0",
+    "Cython>=0.29.21,<3.0",
+    "pybind11>=2.4.3,<2.10.0",
+    "pythran>=0.9.12,<0.12.0",
```

Comment:
Can we up this to `>=0.9.12,<0.12.0` like I just did in `main`? Clang 13 requires 0.11.0, and I've seen multiple issues about that now.

Reply:
Isn't that what I did already?

Reply:
Sorry, I think I meant `>=0.11.0,<0.12.0`. Doesn't matter much though; the range includes the fix for Clang 13, and users are mostly going to get the latest version anyway. 0.9.12 and 0.10.x are good for everything else. 

Resolving.